frontend/public/components/utils/_skeleton-screen.scss

```diff
@@ -94,7 +94,7 @@ $skeleton-tile-desc-line-bone: linear-gradient(#fff $skeleton-tile-desc-line-hei
   align-content: flex-start;
   display: flex;
   flex-wrap: wrap;
-  margin: 117px 6px 6px 0;
+  margin: 100px ($grid-gutter-width / 2) 0 0;
```

Comment:
The skeleton cards now render higher up the page than the cards.

![JgCmmYxHxa](https://user-images.githubusercontent.com/895728/73778428-33816180-4759-11ea-984c-f84664c31722.gif)

Reply:
This is a good catch; I'll fix this.